### ISSUE TITLE
Is it possible to load view controller from storyboard(not XIB)?

### ISSUE THREAD

**Issue by datomnurdin** [uacaps/PageMenu#102]
Is it possible to load view controller from storyboard(not XIB)? This line of code only for XIB

```
var controller : UIViewController = UIViewController(nibName: "SnapNShareViewController", bundle: nil)
```

Please advice. Thank you.

**Comment by rickytan** [comment]
controller = [self.storyboard instantiateViewControllerWithIdentifier:@"Your Controller ID"]

**Comment by courir123** [comment]
yes it is. You have to change the Storyboard ID. Here is my code to load from storyboard and not nib:

       let storyboard = UIStoryboard(name: "Main", bundle: NSBundle.mainBundle())
        
        let controller2 = storyboard.instantiateViewControllerWithIdentifier("second") as! secondViewController

where "second" is the name and storyboard ID of my view controller that I was loading from, and secondViewController is its corresponding UIViewController class.

**Comment by venkatesh613** [comment]
after doing like this not working properly
